src/ui/MobileMenu/ReactedMembersBottomSheet.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { ReactElement, useMemo, useState } from 'react';
+import { EmojiContainer } from '@sendbird/chat';
+import { GroupChannel, Member } from '@sendbird/chat/groupChannel';
+import { FileMessage, Reaction, UserMessage } from '@sendbird/chat/message';
+
+import './mobile-menu-reacted-members.scss';
+
+import BottomSheet from '../BottomSheet';
+import { getEmojiUrl } from '../../utils';
+import ImageRenderer from '../ImageRenderer';
+import Icon, { IconColors, IconTypes } from '../Icon';
+import Label, { LabelColors, LabelTypography } from '../Label';
+import UserListItem from '../UserListItem';
+
+export interface ReactedMembersBottomSheetProps {
+  message: UserMessage | FileMessage;
+  channel: GroupChannel;
+  emojiKey: string;
+  hideMenu: () => void;
+  emojiContainer?: EmojiContainer;
+}
+
+export const ReactedMembersBottomSheet = ({
+  message,
+  channel,
+  emojiKey = '',
+  hideMenu,
+  emojiContainer,
+}: ReactedMembersBottomSheetProps): ReactElement => {
+  const { members = [] } = channel;
+  const [selectedEmoji, setSelectedEmoji] = useState(emojiKey);
+
+  const ReactorList = useMemo(() => {
+    const memberList = message.reactions?.find(reaction => reaction.key === selectedEmoji)
+      ?.userIds.map((userId) => members.find((member) => member.userId === userId) ?? null)
+      .filter((member) => member !== null) as Array<Member>;
```

Comment:
nit; 
Can we have a short comment here to describe what's happening inside? It looks a bit hard to understand only with the code